CDCgov/cdc-open-viz
# Enhancement [DEV 9860] Improve map spacing

## [DEV-9860](https://websupport-cdc.msappproxy.net/browse/DEV-9860)

The OPHDST team would like to improve the spacing between elements (legend, data table, subtext, etc.) that appear below maps. We previously did this for charts and would like to take a pass for maps.

### NOTE
Territories will be handled in a separate PR

## Testing Steps

1. Create or view map with legend
2. Observe there are no large conspicuous gaps

## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)

### Before
<img width="511" alt="Screenshot 2024-11-20 at 2 47 01 PM" src="https://github.com/user-attachments/assets/86917acb-e53e-4f89-a7dd-78c23cd73289">

## After
<img width="328" alt="Screenshot 2024-11-21 at 10 03 04 AM" src="https://github.com/user-attachments/assets/8f86b60a-e412-40c6-b880-3341a04fd21d">
